packages/core-products/src/module/configureProductsModule.ts

```diff
@@ -15,6 +15,10 @@ import {
   buildSortOptions,
 } from '@unchainedshop/utils';
 import { SortDirection, SortOption } from '@unchainedshop/types/api';
+
+import { configureEnrollmentsModule } from '@unchainedshop/core-enrollments';
+import { configureQuotationsModule } from '@unchainedshop/core-quotations';
```

Comment:
this is not allowed by design (core module can't import from other core modules), find another way please

Reply:
see removeProductService.ts for reference